sched/definitions.go

```diff
@@ -17,3 +17,33 @@ type Task struct {
 	Command    []string // Argv to run
 	SnapshotId string
 }
+
+type JobState int
+
+const (
+	// An unambiguous 0-value.
+	UNKNOWN JobState = iota
```

Comment:
I did a poor job of commenting and that made the design unexplained.

I think that once a client has submitted to the Scoot Cloud Exec API, it needs the Cloud Exec API to be able to say what the status is. If jobs are invisible, that's an unworkable design, because clients can't make sense of it.

Fortunately, I think it's easy for to support this API!